api/tests/views_tests.py

```diff
@@ -571,3 +573,48 @@ def test_fxa_responds_with_no_fxa_uid_returns_404_and_cache_returns_404(self):
         assert response.status_code == 404
         assert response.json()["detail"] == "FXA did not return an FXA UID."
         assert responses.assert_call_count(self.fxa_verify_path, 1) is True
+
+
+def _setup_client(token):
+    client = APIClient()
+    client.credentials(HTTP_AUTHORIZATION=f"Bearer {token}")
+    return client
+
+
+@pytest.mark.django_db
+@responses.activate()
+def test_duplicate_email_logs_details_for_debugging(caplog: pytest.LogCaptureFixture):
+    caplog.set_level(logging.ERROR)
+    uid = "relay-user-fxa-uid"
+    email = "user@email.com"
+    baker.make(EmailAddress, email=email)
+    user_token = "user-123"
+    client = _setup_client(user_token)
+    now_time = int(datetime.now().timestamp())
+    # Note: FXA iat and exp are timestamps in *milliseconds*
+    exp_time = (now_time + 60 * 60) * 1000
+    _setup_fxa_response(200, {"active": True, "sub": uid, "exp": exp_time})
+    # setup fxa profile reponse
+    profile_json = {
+        "email": email,
+        "amrValues": ["pwd", "email"],
+        "twoFactorAuthentication": False,
+        "metricsEnabled": True,
+        "uid": uid,
+        "avatar": "https://profile.stage.mozaws.net/v1/avatar/t",
+        "avatarDefault": False,
+    }
+    responses.add(
+        responses.GET,
+        FXA_PROFILE_URL,
+        status=200,
+        json=profile_json,
+    )
+
+    response = client.post("/api/v1/terms-accepted-user/")
+
+    assert response.status_code == 500
+    (rec1,) = caplog.records
+    assert "socialaccount_signup" in rec1.message
+    assert "fxa_uid" in rec1.message
+    assert "social_login.state" in rec1.message
```

Comment:
Try this:
```suggestion
    assert rec1.fxa_uid == uid
    assert getattr(rec1, 'social_login.state') == {}
```

The `getattr` is needed because you used a period in the name, I suggest you don't do that.